test/runner/requirements/units.txt

```diff
@@ -38,3 +38,6 @@ pexpect
 # requirement for the linode module
 linode-python # APIv3
 linode_api4 ; python_version > '2.6' # APIv4
+
+#requirment for kubevirt module
+openshift > 0.8.2
```

Comment:
The version requirement needs to be in the constraints.txt file, and should include a comment explaining why the specified version is required.